### ISSUE TITLE
Is this open source

### ISSUE THREAD

**Issue by jdcdanguk** [Nilsen84/lcqt2#66]
I would like to know if LCQT is open source. I am just curious for anyone wondering.

**Comment by aryananumula** [comment]
Yes, it is. Everything provided in the repository is the actual source files, not the compiled and packed files.
